test/json.js

```diff
@@ -595,6 +605,16 @@ describe('bodyParser.json()', function () {
       test.expect(200, '{"name":"论"}', done)
     })
 
+    if (hasBrotliSupport) {
```

Comment:
Instead of excluding the test, have the test always be there but show up as skipped in the mocha output when the support is missing.